readme.md

```diff
@@ -330,6 +330,9 @@
 - [Data Engineering](https://github.com/igorbarinov/awesome-data-engineering#readme)
 - [Streaming](https://github.com/manuzhang/awesome-streaming#readme)
 - [Apache Spark](https://github.com/awesome-spark/awesome-spark#readme) - Unified engine for large-scale data processing.
+- [Splunk](https://github.com/sduff/awesome-splunk) - Multiple products for searching, monitoring, and analyzing machine-generated big data in real-time.
```

Comment:
I'd say it'd be better to have a main list for splunk and link to the "enterprise security" and "service intellince" from the main one. However since you're using indentation, I'd say that it's 🆗